### PR TITLE
testing/deluge: Update to 1.3.12

### PR DESCRIPTION
**Pull request by AmatCoder**

### CODE REVIEW

**Comment by AmatCoder** [comment]
Travis build fails because #80 needs be merge first.

**Comment by jirutka** [comment]
Please remove whole `prepare()` function. When not defined, then abuild uses `default_prepare` that does exactly the same thing (applying patches).

And you may also rename `_builddir` to `builddir` to be consistent with recent changes in abuild tool.

**Comment by jirutka** [comment]
Please squash commits in this PR into a single commit.

File: testing/deluge/APKBUILD [hunk 1]
```diff
@@ -16,17 +16,7 @@ install=""
...
-	for i in $source; do
-		case $i in
-		*.patch) msg $i; patch -p1 -i "$srcdir"/$i || return 1;;
-		esac
-	done
-}
-
+builddir="$srcdir"/$pkgname-$pkgver
 build() {
 	cd "$_builddir"
```
Comment (jirutka) [inline testing/deluge/APKBUILD]: You must obviously also rename all occurrences of `$_builddir` to `$builddir`.

**Comment by AmatCoder** [comment]
Done. Thanks for suggestions.

**Comment by ncopa** [comment]
applied. thanks

**Pull request closed without merging**